File: Examen  Final/Final/app/src/main/java/com/example/afinal/Doctor.kt
package com.example.afinal

 class Doctor (
    var id: Int,
    var createdAt: Long,
    var updatedAt: Long,
    var nombre:String,
    var edad:String,
    var tipoSangre:String,
    var peso:String,

    var paciente:String) {

     override fun toString(): String {
         return "Nombre: ${nombre}, edad:${edad}"
     }
 }

File: Examen  Final-Doctor Paciente/Back/hospital/api/models/Doctor.js
/**
 * Doctor.js
 *
 * @description :: A model definition represents a database table/collection.
 * @docs        :: https://sailsjs.com/docs/concepts/models-and-orm/models
 */

module.exports = {



    attributes: {

      nombre:{
        type:'string',
        required:true, // Por defecto es false
      },
      edad:{
        type: 'string',
  
      },
      tipoSangre:{
        type:'string',
  
      },
      peso:{
        type:'string',
  
      },
  
     paciente:{
        type:'string',
      }
  },


}

